Google.Cloud.EntityFrameworkCore.Spanner/Migrations/SpannerMigrationsSqlGenerator.cs

```diff
@@ -45,6 +45,67 @@ public SpannerMigrationsSqlGenerator(MigrationsSqlGeneratorDependencies dependen
         {
         }
 
+        protected override void Generate(
+            AlterColumnOperation operation,
+            IModel model,
+            MigrationCommandListBuilder builder)
+        {
+            builder
+               .Append("ALTER TABLE ")
+               .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(operation.Table, operation.Schema))
+               .Append(" ALTER COLUMN ");
+
+            var commitTimestampAnnotation = operation.FindAnnotation(SpannerAnnotationNames.UpdateCommitTimestamp);
+            if (commitTimestampAnnotation != null)
+            {
+                if ((SpannerUpdateCommitTimestamp)commitTimestampAnnotation.Value != SpannerUpdateCommitTimestamp.Never)
+                {
+                    builder
+                        .Append(operation.Name)
+                        .Append(" SET OPTIONS (allow_commit_timestamp=true) ");
```

Comment:
Does this work if the option is not really being altered. That is, if the current value is already `true`, and the alter column operation is only for making the column nullable/not nullable for example? Same also applies if the value is already `null` and the alter operation is only changing some other property of the column?